### PR TITLE
Adding momentable NFT details for Alchemy public api integrations

### PR DESCRIPTION
**Pull request by palani-kochatech**

### CODE REVIEW

File: src/cadence/testnet/getNFTs.cdc [hunk 1]
```diff
@@ -2131,4 +2133,40 @@ pub fun getRacingTimeNFT(owner: PublicAccount, id: UInt64): NFTData? {
...
+    return NFTData(
+        contract: contract,
+        id: nft!.id,
+        uuid: nft!.uuid,
+        title: nft!.name,
+        description: nft!.description,
+        external_domain_view_url: nil,
+        token_uri: nil,
+        media: [NFTMedia(uri: ipfsURL, mimetype: "image")],
+        metadata: nft!.getTraits(),
```
Comment (niveda-krish) [inline src/cadence/testnet/getNFTs.cdc:2170]: Thanks for submitting this PR! We just updated the repo to store metadata as a {String: String?} dictionary, could you pull from the latest main branch and merge?

Comment (niveda-krish) [reply src/cadence/testnet/getNFTs.cdc:2170]: Looks great otherwise!

Comment (palani-kochatech) [reply src/cadence/testnet/getNFTs.cdc:2170]: Sure Thanks. Will check it out.

Comment (palani-kochatech) [reply src/cadence/testnet/getNFTs.cdc:2251]: Hi @niveda-krish We have a Nested metadata structure as shared below. Not sure why you moved from String: AnyStruct to String: String. I am trying a few other ways to concatenate those nested objects but not sure if that might be rendered correctly. Do you have any suggestions are is there a way Alchemy can support AnyStruct ?

`	{"Hallowed Accessory": {
		"rarity": "12.69%",
		"primaryPower": "Wisdom 3d4",
		"additionalData": "Darkness",
		"name": "Gilded Eye",
		"secondaryPower": "Constitution 2d6"
	},
	"Kingdom": {
		"name": "The Seventh Kingdom",
		"rarity": "15.22%"
	}
}`

Please let me know.

Comment (palani-kochatech) [reply src/cadence/testnet/getNFTs.cdc]: For now, I did some parsing of the above data to match Alchemy's metadata structure. However, I still believe the AnyStruct could have been a better option. Please suggest.

Comment (palani-kochatech) [reply src/cadence/testnet/getNFTs.cdc]: @niveda-krish  Updated the PR with the new metadata structure. Kindly review and merge

**Review by niveda-krish — APPROVED** [review]

File: src/cadence/testnet/getNFTs.cdc (hunk 1 above)
Comment (niveda-krish) [reply src/cadence/testnet/getNFTs.cdc]: Hey @palani-kochatech, thanks for updating the contract. We are working with the Flow team on a new metadata standard. Until we have that in place, we expect to support only {String : String?} metadata to have consistency across all contracts. One way to go about serializing AnyStruct data in your case would be to iterate and store all the nested key-value pairs in the metadata struct!

Comment (niveda-krish) [reply src/cadence/testnet/getNFTs.cdc]: Your updates look great! Could you resolve merge conflicts with latest main branch?

**Comment by palani-kochatech** [comment]
@niveda-krish Can you please merge this PR! Thanks in advance.

**Pull request merged**